domains/tukdanai-book.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "BookIILoy",
+           "email": "tudanai.book@mail.kmutt.ac.th",
+           "discord": "370091250637996033"
+        },
+    
+        "record": {
+            "TXT": "_vercel"
```

Comment:
TXT doesn't do anything (Also as far as I know we don't support Vercel at the moment)